components/form/style/index.less

```diff
@@ -3,6 +3,452 @@
 @import '../../input/style/mixin';
 @import '../../button/style/mixin';
 @import '../../grid/style/mixin';
+@import './components';
+@import './inline';
+@import './horizontal';
+@import './vertical';
 @import './mixin';
 
 @form-prefix-cls: ~'@{ant-prefix}-form';
+@form-item-prefix-cls: ~'@{form-prefix-cls}-item';
+@form-font-height: ceil(@font-size-base * @line-height-base);
+@form-label-margin-top: floor((@input-height-base - @form-font-height) / 2);
+
+.@{form-prefix-cls} {
+  .reset-component;
+  .reset-form;
+
+  .@{form-prefix-cls}-text {
+    display: inline-block;
+    margin-top: @form-label-margin-top;
+    padding-right: 8px;
+  }
+}
+
+// ================================================================
+// =                             Item                             =
+// ================================================================
+.@{form-item-prefix-cls} {
+  .reset-component;
+
+  margin-bottom: @form-item-margin-bottom;
+  vertical-align: top;
+
+  &-with-help {
+    margin-bottom: 0;
+  }
+
+  // ==============================================================
+  // =                            Label                           =
+  // ==============================================================
+  &-label {
+    display: inline-block;
+    overflow: hidden;
+    white-space: nowrap;
+    text-align: right;
+    vertical-align: middle;
+
+    &-left {
+      text-align: left;
+    }
+
+    > label {
+      position: relative;
+      display: inline-block;
+      margin-top: @form-label-margin-top;
```

Comment:
That line created a misalignment when we use label and input with diff sizes, for ex:

```javascript
<Input size="large"/>
```

Reply:
Label position is a fixed value instead of related vertical center of the Item since some component like TextArea (or Transfer or Tree or some else) occupy the more height of it.

For the size diff component, maybe provides some other prop to handle (still in thinking).

ref: https://codesandbox.io/s/hungry-driscoll-zc9cd